mkdocs.yml

```diff
@@ -47,5 +47,6 @@ extra_css:
   - stylesheets/extra.css
 
 extra_javascript:
+  - javascripts/mathjax.js
```

Comment:
Where was this plugin used? 👀

Reply:
Uhh just now realized.. I did this an attempt to fix an equation in lesson 3.6 but then I fixed it by putting the equation in an align env.. this change was totally unnecessary. Reverting this!